res/css/views/right_panel/_TimelineCard.scss

```diff
@@ -90,7 +90,8 @@ limitations under the License.
         }
```

Comment:
Let's add a before/after screenshot of the same area to the issue description for easy comparison.

Reply:
ok, please wait for a moment..